src/include/homestore/replication/repl_decls.h

```diff
@@ -65,6 +65,15 @@ using remote_blkid_list_t = folly::small_vector< RemoteBlkId, 4 >;
 using replica_id_t = uuid_t;
 using group_id_t = uuid_t;
 
+struct peer_info {
```

Comment:
is there a reason we can't use the same structure defined in nuraft_mes: https://github.com/eBay/nuraft_mesg/blob/main/include/nuraft_mesg/mesg_state_mgr.hpp#L31

Reply:
We are not exposing nuraft_mesg to the upper layer, so we probably have to redefine here.

Reply:
My thinking is 
1. the structure field (mostly `last_log_idx_`) is too implementation(raft) specific,  as a generic API for all repl-dev I am trying to use more neutral naming.
2. HS/HO we use replica_id_t,  translate it from string to replica_id_t make it more consistent from the view of HO.